### ISSUE TITLE
problem with using cookies and the /admin page

### ISSUE THREAD

**Issue by sbernier1** [flavors/django-graphql-jwt#227]
Hi,

I was using localStorage for my JWT authentication. It worked fine but I decided to switch to cookies since they're apparently more secure when all the other safety measures are in place. I'm encountering problems now when I try to use the /admin page and my frontend app. They work fine by themselves but using them together shows weird behaviors.

I have a pretty basic setup; my backend is on localhost:8000 and my frontend app is on localhost:8080.

problems I'm encountering: 

1. Login through my app first: I can use my app normally, the cookie gets set and context.user gives the user described in the cookie. However, I can't access the django admin page after this: going to localhost:8000/admin redirects me to the sign in page (which is fine), and when I try to login I always get redirected to the sign in page again, without any errors and I receive a Set-Cookie sessionId (even if using a bad username/password combination or the same user as my app or my superuser, I suspect it always uses the user from the cookie. Not sure why it won't let me through the login page though). logging in my app with my superuser gives the same result. With local storage and the Authorization header I could login with any user in the admin page and it would not have any impact whatsoever on my app.

2. Login through the admin page first: I can access the admin page normally. When I go to my app, I have to login (expected). When I do, login works and the cookie gets set BUT the user context of all my requests is my admin page user. It used to give me the app user.

**Comment by chidimo** [comment]
I'm having same issue. I have a multi-tenant app that are being differentiated by domain. The problem is when I log in to two different domains on the same browser, the users get confused when I make a call to the backend.

**Comment by abbasegbeyemi** [comment]
Facing the exact same issue as well. In my case, the front end is supposed to be on a different domain from the back end. When I am not logged in to the admin site, things work as expected. However when I am logged into the admin site, the authorization header sent with the request is not respected at all, and the context is always the user logged in to Django admin.

Any suggestions on how to deal with this?

**Comment by mongkok** [comment]
Hey guys,
The issue is fixed with [PR #283](https://github.com/flavors/django-graphql-jwt/pull/283).
Github actions are down today, so I will create a new release tomorrow.

In the meantime here is a hotfix:
```py
AUTHENTICATION_BACKENDS = [
    'django.contrib.auth.backends.ModelBackend',
    'graphql_jwt.backends.JSONWebTokenBackend',
]
```
..just reorder the authentication backends.

Thanks @sbernier1 for reporting the issue and for this detailed description.
I apologize for the delay in my response.
